op-scim-bridge/Makefile

```diff
@@ -12,56 +12,55 @@ APP_GCS_PATH ?= $(GCS_URL)/$(APP_ID)/$(TAG)
 CHART_NAME ?= op-scim-bridge
 NAME ?= op-scim-bridge
 APP_PARAMETERS ?= { \
-  "name": "$(NAME)", \
-  "namespace": "$(NAMESPACE)" \
+	"name": "$(NAME)", \
+	"namespace": "$(NAMESPACE)" \
 }
 APP_TEST_PARAMETERS ?= {}
 
 include ../tools/app_v2.Makefile
 
 app/build:: .build/op-scim/deployer \
-            .build/op-scim/op-scim-redis \
-            .build/op-scim/op-scim-bridge
-
+			.build/op-scim/op-scim-redis \
+			.build/op-scim/op-scim-bridge
 
 .build/op-scim: | .build
 	mkdir -p "$@"
 
-
-.build/op-scim/deployer: deployer/* \
-								$(shell find chart -type f) \
-                           		schema.yaml \
-                           		.build/var/APP_DEPLOYER_IMAGE \
-								.build/var/REGISTRY \
-								.build/var/TAG \
-								.build/var/PUBLIC_TAG \
-								| .build/op-scim
+.build/op-scim/deployer: .build/var/APP_DEPLOYER_IMAGE \
+						 $(shell find chart -type f) \
+						 .build/var/MARKETPLACE_TOOLS_TAG \
+						 .build/var/REGISTRY \
+						 .build/var/TAG \
+						 apptest/deployer/* \
+						 deployer/* \
+						 schema.yaml \
+						 | .build/op-scim
+	$(call print_target, $@)
 	docker build \
-	    --build-arg REGISTRY="$(REGISTRY)/op-scim-bridge" \
-	    --build-arg TAG="$(TAG)" \
-	    --build-arg PUBLIC_TAG="$(PUBLIC_TAG)" \
```

Comment:
Can you explain why we are removing the `PUBLIC_TAG`? Do we not need it when building the deployer?

The usage in the past has been `PUBLIC_TAG=$version_I_want_to_use $TAG=$my_development_tag_to_tag`

Correct me if I'm wrong, the current setup will only allow us to tag production releases of the deployer, not development releases. 

I note you still use the $PUBLIC_TAG below in the `build/op-scim/op-scim-bridge` command.

What am I missing?

Reply:
The `deployer` image actually uses the `deployer/Dockerfile` to be built, and one thing that I noticed is that the `Dockerfile` does not actually ingest the `PUBLIC_TAG` environment variable at any point during the image being built. This means that `PUBLIC_TAG` is simply not used during that command, so I removed it to be more consistent. I actually did not make many modifications to the `Dockerfile`, and its lack of `PUBLIC_TAG` isn't something I decided on. We could talk about adding it in, but that's simply the way it has been until now.